packages/core/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -71,19 +80,22 @@ export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
         role="progressbar"
         aria-valuemax={max}
         aria-valuemin={min}
-        aria-valuenow={Math.round(value)}
+        aria-valuenow={isIndeterminate ? undefined : Math.round(value)}
```

Comment:
Correct me if I am wrong ?
if I am a determinate progress bar then the  aria-valuenow changes, so I get non-visual feedback of the state change. If I am indeterminate, then there is nothing here that gives us any non-visual feedback ?
The progress bar could be active (indeterminate) but the non-visual feedback would be the same.
The associated label does not use labelledBy so, it has no association.
This feels like an accessibility issue?

Reply:
I can change it to match checkbox but it will deviate from what we have in UITK, can I double check we are happy with that @mark-tate ?